Containerfile

```diff
@@ -37,125 +37,79 @@ COPY system_files/desktop/shared system_files/desktop/${BASE_IMAGE_NAME} /
 
 # Update packages that commonly cause build issues
 RUN --mount=type=cache,dst=/var/cache/rpm-ostree \
```

Comment:
We've been using DNF 5 in PlaytronOS for a few months now and, yes, it's amazing!

One of the great things is that it takes care of dependencies. It's okay if a package is slightly out-of-date, DNF will automatically deal with that by updating the package(s) as necessary. rpm-ostree had a hard time because it didn't know how to update base packages. You should be able to remove this entire "Update packages that commonly cause build issues" block.